### PR TITLE
Feature/How it wroks

### PR DESCRIPTION
**Pull request by Sslim123**
## Description

Write a summary of your change. Explain why you have made it.

## Related to

Make sure you include the issue number with a hash sign # so Github can link this PR to the right issue, like this:

Fixes #1

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have carefully reviewed my own code
- [ ] I have commented my code
- [ ] I have updated any documentation

### CODE REVIEW

File: client/src/pages/Home/Home.js [hunk 1]
```diff
@@ -5,13 +5,14 @@ import Footer from "../component/footer/Footer";
 import "./Home.css";
 import Mission from "../component/Mission/Mission";
 import ContactUs from "../component/ContactUs/ContactUs";
+import HowIsWORK from "../component/HowItIsWork/HowIsWork";
```
Comment (shieldo) [inline client/src/pages/Home/Home.js]: I'd go through everywhere that the name of this page "How it works" is being referenced and make sure that it is referred to in the same way: i.e. `HowItWorks` here, HowItWorks.js, howitworks.css etc etc. It really helps other developers if naming is consistent.

**Review by shieldo — COMMENTED** [review]
As well as comments about consistent naming, do make sure that image files are named well and reflect what the images are of. These images are generally icons, so it's a good idea to make up a common convention for the naming, like 'icon-laptop.svg' and the like.

Do also look at the homepage in different sizes on your local machine - this new area of the homepage acts differently to the other areas of the homepage, and areas end up sitting on top of each other. If you think that some of the other areas of the page need attention, the discuss that with the rest of your team and work out what you are going to do in this particular change.

File: client/src/pages/Home/Home.js (hunk 1 above)
Comment (Sslim123) [reply client/src/pages/Home/Home.js]: yes, i should do what you said,  thank for advices

Comment (Sslim123) [reply client/src/pages/Home/Home.js]: i saw this different on the my layout page,my work is fine, it display well this different from Abed's work, we got the issue, i discuss this with Abed we decide to get solution for it

**Comment by shieldo** [comment]
I think the naming could still be more consistent in a few places but thanks, this is better. Thanks also for talking within the team about next steps in making the homepage right.

**Pull request merged**